Makefile

```diff
@@ -71,24 +71,14 @@ install-ocamlformat:
 dev-depext:
 	opam depext -y $(TEST_DEPS)
 
-.PHONY: melange
-melange:
-	opam pin add -n melange-compiler-libs.dev https://github.com/melange-re/melange-compiler-libs.git#575ac4c24b296ea897821f9aaee1146ff258c704
-	opam pin add -n melange.dev https://github.com/melange-re/melange.git#60eabf03d60b1e71f8d5aae72d24d7a1dfcf5bf6
-	opam pin add -n rescript-syntax.dev https://github.com/melange-re/melange.git#60eabf03d60b1e71f8d5aae72d24d7a1dfcf5bf6
-
 .PHONY: dev-deps
-dev-deps: melange
+dev-deps:
 	opam install -y $(TEST_DEPS)
 
 .PHONY: coverage-deps
 coverage-deps:
 	opam install -y bisect_ppx
 
-.PHONY: dev-deps-sans-melange
-dev-deps-sans-melange:
```

Comment:
This command is used here

https://github.com/ocaml/dune/blob/bd1193de7b5b59770d01d0a3f43e6936c6271a16/ci/build-test.sh#L120

Same with `test-all-sans-melange` below.

Reply:
I updated this by following the comments. But it's not used since we only use that file on windows.